ic-stable-structures/src/structure/cache/unbounded.rs

```diff
@@ -33,6 +35,17 @@ where
             cache: SyncLruCache::new(max_cache_items),
         }
     }
+
+    /// Iterator for all stored key-value pairs.
+    pub fn iter(&self) -> StableUnboundedIter<'_, K, V, M> {
```

Comment:
The iterator was omitted from the cache because it bypasses the cache completely

Reply:
Yes it does. But:
1. I don't think there is a way to implement iterator that uses cached values to speedup the iteration.
2. LRU cache won't speedup the iteration over the whole collection by it's nature. Moreover in case when we put the values over which we are iterating to the cache we most probably would just fill it with irrelevant values.

I propose either to leave the implementation as is or just expose `inner` so that it can be iterated explicitly.

Reply:
Replaced with `inner()` method